### PR TITLE
tidb-binlog: update kafka-max-message-size configuration

### PR DESCRIPTION
**Pull request by lichunzhu**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

update kafka-max-message-size configuration

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v5.3 (TiDB 5.3 versions) [This feature is cherry-pick but not released below this version]
- [x] v5.2 (TiDB 5.2 versions) 
- [x] v5.1 (TiDB 5.1 versions)
- [x] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): https://github.com/pingcap/tidb-binlog/pull/1039

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-binlog/deploy-tidb-binlog.md [hunk 1]
```diff
@@ -358,6 +358,8 @@ Drainer="192.168.0.13"
         # kafka-addrs = "127.0.0.1:9092"
         # kafka-version = "0.8.2.0"
         # kafka-max-messages = 1024
+        # 配置单条 kafka 消息的最大 size（单位为 Byte），默认为 1G，最大配置 2G
```
Comment (TomShawn) [inline tidb-binlog/deploy-tidb-binlog.md:361]:
```suggestion
        # 配置单条 kafka 消息的最大 size（单位为 Byte），默认为 1 GB，最大配置 2 GB
```

File: tidb-binlog/deploy-tidb-binlog.md [hunk 2]
```diff
@@ -358,6 +358,8 @@ Drainer="192.168.0.13"
         # kafka-addrs = "127.0.0.1:9092"
         # kafka-version = "0.8.2.0"
         # kafka-max-messages = 1024
+        # 配置单条 Kafka 消息的最大 size（单位为 Byte），默认为 1 GB，最大可配置为 2 GB
```
Comment (glorv) [inline tidb-binlog/deploy-tidb-binlog.md:361]:
```suggestion
        # 配置单条 Kafka 消息的最大 size（单位为 Byte），默认为 1 GiB，最大可配置为 2 GiB
```

**Review by glorv — COMMENTED** [review]
rest LGTM

File: tidb-binlog/deploy-tidb-binlog.md (hunk 2 above)
Comment (glorv) [reply tidb-binlog/deploy-tidb-binlog.md:361]: BTW, 最好给 `kafka-max-messages` 也加个注释

**Review by glorv — COMMENTED** [review]
LGTM

File: tidb-binlog/deploy-tidb-binlog.md [hunk 3]
```diff
@@ -357,7 +357,10 @@ Drainer="192.168.0.13"
         # zookeeper-addrs = "127.0.0.1:2181"
         # kafka-addrs = "127.0.0.1:9092"
         # kafka-version = "0.8.2.0"
+        # 配置单条 broker request 中的最大 message 数（即 binlog 数），不配置或配置小于等于 0 的值时会使用默认值 1024
```
Comment (Liuxiaozhen12) [inline tidb-binlog/deploy-tidb-binlog.md:360]:
```suggestion
        # 配置单条 broker request 中的最大 message 数（即 binlog 数），不配置或配置小于等于 0 时会使用默认值 1024
```

**Review by Liuxiaozhen12 — APPROVED** [review]

**Review by TomShawn — APPROVED** [review]

**Pull request merged**